proto/lndrpc.proto

```diff
@@ -1625,10 +1625,10 @@ message Invoice {
     The hex-encoded preimage (32 byte) which will allow settling an incoming
     HTLC payable to this preimage
     */
-    bytes r_preimage = 3 [json_name = "r_preimage"];
+    bytes rPreimage = 3 [json_name = "rPreimage"];
```

Comment:
FYI we're using snake_case for the proto definition, I was thinking of only changing the names internally.